src/GraphQLParser/ROM.cs

```diff
@@ -4,7 +4,11 @@ namespace GraphQLParser;
 
 /// <summary>
 /// A wrapper around ReadOnlyMemory{char} allowing you to use simple syntax when working with it.
-/// <br/>
+/// <br/><br/>
+/// This struct is designed to unify work with objects of <see cref="ReadOnlyMemory{T}"/>,
+/// <see cref="ReadOnlySpan{T}"/> ans <see cref="string"/>. For example, you can directly
```

Comment:
@Shane32 I added these notes to make ROM usage by other projects more understandable.